types/v1/util.go

```diff
@@ -1220,3 +1221,38 @@ func OSGetEnv(envKey string, profileMap map[string]string) string {
 
 	return val
 }
+
+// ParseAndSubstract parses the string and then returns the the substracted
+// values and error if any.
+func ParseAndSubstract(initial, final string) (float64, error) {
```

Comment:
OPTIONAL - Though this seems to be a utility function it might make more sense to have it as a method of Metrics struct itself. Or atleast moved to the file which defines Metrics.

Reply:
roger that.